src/firebolt/client/client.py

```diff
@@ -30,34 +32,51 @@ def __init__(
         super().__init__(*args, auth=auth, **kwargs)
 
     def _build_auth(self, auth: _types.AuthTypes) -> Optional[Auth]:
+        """Create Auth objects based on auth provided.
+
+        Overrides ``httpx.Client._build_auth``
+
+        Args:
+            auth (_types.AuthTypes): Provided auth
+
+        Returns:
+            Optional[Auth]: Auth object
+
+        Raises:
+            TypeError: Auth argument has unsupported type
+        """
         if auth is None or isinstance(auth, Auth):
             return auth
-        elif isinstance(auth, tuple):
+        if isinstance(auth, tuple):
             return Auth(
                 username=str(auth[0]),
                 password=str(auth[1]),
                 api_endpoint=self._api_endpoint,
             )
-        else:
-            raise TypeError(f'Invalid "auth" argument: {auth!r}')
+        raise TypeError(f'Invalid "auth" argument: {auth!r}')
 
 
 class Client(FireboltClientMixin, HttpxClient):
+    """An HTTP client, based on httpx.Client.
 
-    """
-    An http client, based on httpx.Client, that handles the authentication
-    for Firebolt database.
-
+    Handles the authentication for Firebolt database.
     Authentication can be passed through auth keyword as a tuple or as a
     FireboltAuth instance
-
-    httpx.Client:
-
-    + (HttpxClient.__doc__ or "")
     """
 
     @cached_property
     def account_id(self) -> str:
+        """User account id.
+
+        If account_name is provided, returns it's id.
```

Comment:
I'd rephrase it to something like "If account_name was provided during the instantiation of this class...".
This way we avoid confusion that the docstring talks about a parameter that was removed from `account_id()` and the comment was not updated.

Reply:
Makes sense!